docs/UserGuide.adoc

```diff
@@ -854,7 +861,8 @@ Adds a new session with:
 ** `START_TIME` at 14:00 in `24hr` format
 ** `END_TIME` at 16:00 in `24hr` format
 ** `DATE` on 2020-06-20 in `yyyy-MM-dd` format
-** `WEEK` recurring period of 2 weeks
+** `WEEK` recurring period of 2 weeks: Once this session has been marked as done, a new session will
+be created with `DATE d/2020-20-20` (2 weeks later from the date of the original session)
```

Comment:
Date here is wrong! there is no 20th month lol. I believe the date would be 2020-06-27

Reply:
Thanks for pointing it out! should be 2020-07-04 actually since its 2 weeks later